tests/gpflow/utilities/test_traversal.py

```diff
@@ -209,14 +209,15 @@ def create_model():
 # Note: we use grid format here because we have a double reference to the same variable
 # which does not render nicely in the table formatting.
 example_tf_keras_model = """\
-+-------------------------+------------------+-------------+---------+-------------+-----------+---------+----------+\n\
-| name                    | class            | transform   | prior   | trainable   | shape     | dtype   | value    |\n\
-+=========================+==================+=============+=========+=============+===========+=========+==========+\n\
-| C._trainable_weights[0] | ResourceVariable |             |         | True        | (2, 2, 1) | float32 | [[[0.... |\n\
-| C.variable              |                  |             |         |             |           |         |          |\n\
-+-------------------------+------------------+-------------+---------+-------------+-----------+---------+----------+\n\
-| C.param                 | Parameter        | Identity    |         | True        | ()        | float64 | 0.0      |\n\
-+-------------------------+------------------+-------------+---------+-------------+-----------+---------+----------+"""
++-------------------------------+------------------+-------------+---------+-------------+-----------+---------+----------+\n\
+| name                          | class            | transform   | prior   | trainable   | shape     | dtype   | value    |\n\
++===============================+==================+=============+=========+=============+===========+=========+==========+\n\
+| C._trainable_weights[0]       | ResourceVariable |             |         | True        | (2, 2, 1) | float32 | [[[0.... |\n\
+| C.variable                    |                  |             |         |             |           |         |          |\n\
++-------------------------------+------------------+-------------+---------+-------------+-----------+---------+----------+\n\
+| C._self_tracked_trackables[0] | Parameter        | Identity    |         | True        | ()        | float64 | 0.0      |\n\
+| C.param                       |                  |             |         |             |           |         |          |\n\
```

Comment:
How about simply changing the expected string based on the TF version? We've got similar test branching in the tests of clearing the bijector caches.

Reply:
Done.